src/test/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineServicesTest.java

```diff
@@ -61,7 +61,7 @@ public void setTrafficTest() throws ProcessRunnerException {
     appEngineService.setTraffic(configuration);
 
     List<String> args =
-        Arrays.asList("services", "set-traffic", "myService", "--splits", "v1=0.3,v2=0.7",
+        Arrays.asList("services", "set-traffic", "myService", "--splits", "v1=0.3", "v2=0.7",
```

Comment:
Yeah I think so. Or did at least when I first implemented this. I'll check the current state.